issue_01/index.html

```diff
@@ -11,8 +11,9 @@
 
 <body>
     <div>
-        <ul></ul>
+        <ul id="target"></ul>
```

Comment:
IMO
IDs and classes that are the starting point for JavaScript are prefixed with `js-` to clearly indicate that JavaScript is involved.
It is also a good idea to use an ID name that is easy for everyone to understand. This time, why not keep it simple and use ul?
[Reference](https://zenn.dev/haniwaman/articles/bf392f397c8db7341881#javascript%E3%81%A8%E7%B4%90%E3%81%A5%E3%81%8F%E8%A6%81%E7%B4%A0%E3%81%AFjs-%E3%81%AE%E6%8E%A5%E9%A0%AD%E8%BE%9E)
```suggestion
        <ul id="js-ul"></ul>
```

Reply:
Thank you!
You're right, the ID should be easy for everyone to understand.
 I will be aware of this in the future.
[7338b46](https://github.com/yoshikawa-akr/js-lessons/pull/1/commits/7338b462283954623d7f947002b6f7418a9b21c8)